app/models/heads_of_term.rb

```diff
@@ -36,6 +36,10 @@ def any_new_updated_validation_requests?
     validation_requests.requests_created_later(current_review).any? { |validation_request| !validation_request.approved.nil? }
   end
 
+  def create_heads_of_terms_review!
+    reviews.create!(reviewer: Current.user, owner_type: "Heads of Term", owner_id: id, specific_attributes: {"review_type" => "heads_of_term"}, status: "complete")
```

Comment:
I don't think you need the review_type to be set here, because the owner is unique so we'll know it's a heads of terms review, unlike consultation